### PR TITLE
Temporarily disable uploads of client/web LSIF dumps to dogfood.

### PR DESCRIPTION
**Pull request by varungandhi-src**

### CODE REVIEW

File: .github/workflows/lsif-ts.yml [hunk 1]
```diff
@@ -53,7 +53,12 @@ jobs:
           SRC_ENDPOINT: https://sourcegraph.com/
       - name: Upload lsif-tsc dump to Dogfood
         working-directory: ${{ matrix.root }}
```
Comment (eseliger) [inline .github/workflows/lsif-ts.yml:55]: I think you could also use `if: ${{ matrix.root }} == "client/web"` so we don't need to actually spin up anything for this job, but not a blocker

**Review by eseliger — APPROVED** [review]

File: .github/workflows/lsif-ts.yml (hunk 1 above)
Comment (varungandhi-src) [reply .github/workflows/lsif-ts.yml:55]: Good point, I hadn't realized that was an option.

**Comment by Strum355** [comment]
Plan is to keep them uploading to cloud still, I'm assuming for pprof?

**Comment by varungandhi-src** [comment]
Yes, let's run some profiling first and then get rid of that too. I just saw production and the queue there has ballooned to over 1k+ from 500 earlier. 😢

File: .github/workflows/lsif-ts.yml [hunk 2]
```diff
@@ -53,6 +53,9 @@ jobs:
...
+        # Temporarily disable client/web uploads to Dogfood because of long queues
+        # https://github.com/sourcegraph/sourcegraph/issues/30493#issuecomment-1030090306
+        if: ${{ matrix.root }} != "client/web"
```
Comment (varungandhi-src) [inline .github/workflows/lsif-ts.yml:58]: This PR doesn't seem to have worked, I'm still seeing uploads of `client/web` to dogfood.

Comment (varungandhi-src) [reply .github/workflows/lsif-ts.yml:58]: This seems to be down to https://github.com/actions/runner/issues/866

**Pull request merged**